mock/client.go

```diff
@@ -0,0 +1,21 @@
+package mock
+
+import (
+	"net/http"
+
+	recurly "github.com/blacklightcms/go-recurly"
+)
+
+// NewClient sets the unexported fields on the struct and returns a Client.
+func NewClient(httpClient *http.Client) *recurly.Client {
+	if httpClient == nil {
+		httpClient = http.DefaultClient
+	}
+
+	client := recurly.NewClient("a", "b", httpClient)
+
+	client.Subscriptions = &SubscriptionsService{}
+	client.Transactions = &TransactionsService{}
+
```

Comment:
Can you set all of the other services without mocks to nil so they panic until we implement them? Otherwise someone using this mock client will have a mix of mocks and actual api implementations.